client/src/public/css/custom-multiplayer-room-selection-screen.css

```
@@ -0,0 +1,133 @@
+  z-index: var(--z-index-popup-notification);
+  overflow: hidden;
+  width: 100vw;
+  height: 100vh;
+  position: absolute;
+  top: 0;
+  pointer-events: none;
+  background-color: #000000aa;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  pointer-events: all;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

pointer-events is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)